misc/git-hooks/pre-commit

```diff
@@ -89,7 +89,7 @@ function check_check_forbidden_dependencies() {
       return 1
     fi
     find modules/ -name Cargo.toml | grep common/ | while read -r cargo_toml ; do
-      if grep -E "fedimint-" "$cargo_toml" | grep -E -v "fedimint-core|fedimint-api-client|-common|fedimint-logging" >&2 ; then
+      if grep -E "fedimint-" "$cargo_toml" | grep -E -v "fedimint-core|fedimint-api-client|-common|fedimint-logging|fedimint-lightning" >&2 ; then
```

Comment:
@dpc is this the right place to modify this? Comment says its auto-generated by flakebox

Reply:
You're looking for `nix/check-forbidden-deps.sh` @m1sterc001guy

Reply:
Done